src/components/sharedNotebookRenderStrategy.tsx

```diff
@@ -67,27 +67,27 @@ export class SharedNotebookRenderStrategy implements ExpandableNodeRenderStrateg
 			];
 		}
 
-		const sectionGroupRenderStrategies = this.notebook.apiProperties.spSectionGroups.map(sectionGroup => new SectionGroupRenderStrategy(sectionGroup, this.globals));
-		const sectionGroups = sectionGroupRenderStrategies.map(renderStrategy =>
-			!!this.globals.callbacks.onSectionSelected || !!this.globals.callbacks.onPageSelected ?
+		const sectionRenderStrategies = this.notebook.apiProperties.spSections.map(section => new SectionRenderStrategy(section, this.globals));
+		const sections = sectionRenderStrategies.map(renderStrategy =>
+			!!this.globals.callbacks.onPageSelected ?
 				<ExpandableNode
 					expanded={renderStrategy.isExpanded()} node={renderStrategy} globals={this.globals}
 					treeViewId={Constants.TreeView.id} key={renderStrategy.getId()}
 					id={renderStrategy.getId()} level={childrenLevel} ariaSelected={renderStrategy.isAriaSelected()} /> :
 				<LeafNode node={renderStrategy} treeViewId={Constants.TreeView.id} key={renderStrategy.getId()} globals={this.globals}
 					id={renderStrategy.getId()} level={childrenLevel} ariaSelected={renderStrategy.isAriaSelected()} />);
 
-		const sectionRenderStrategies = this.notebook.apiProperties.spSections.map(section => new SectionRenderStrategy(section, this.globals));
-		const sections = sectionRenderStrategies.map(renderStrategy =>
-			!!this.globals.callbacks.onPageSelected ?
+		const sectionGroupRenderStrategies = this.notebook.apiProperties.spSectionGroups.map(sectionGroup => new SectionGroupRenderStrategy(sectionGroup, this.globals));
+		const sectionGroups = sectionGroupRenderStrategies.map(renderStrategy =>
+			!!this.globals.callbacks.onSectionSelected || !!this.globals.callbacks.onPageSelected ?
```

Comment:
The order of these code blocks doesn't matter... just the order of the variables in the array returned by the method. Just FYI.

Reply:
Thought it looks nicer this way :)